src/main/scala/mesosphere/marathon/state/TaskKiller.scala

```diff
@@ -0,0 +1,50 @@
+package mesosphere.marathon.state
```

Comment:
The TaskKiller holds some functionality that is shared in two Resource classes. Does it makes sense to move this class here? 

(I know the package structure we have is far from ideal, but at least it does not deal with state)